cmd/install/assets/hypershift_operator.go

```diff
@@ -633,7 +632,7 @@ func (o HyperShiftOperatorDeployment) Build() *appsv1.Deployment {
 							Command:         []string{"/usr/bin/hypershift-operator"},
 							Args:            []string{"init"},
 							SecurityContext: &corev1.SecurityContext{
-								RunAsUser:              k8sutilspointer.Int64(1000),
+								RunAsUser:              ptr.To(int64(1000)),
```

Comment:
please use generics instead of casting
```suggestion
								RunAsUser:              ptr.To[int64](1000),
```
same comment for the rest